docs/source/38_0.rst

```diff
@@ -0,0 +1,135 @@
+===============
+38.0: Love, Ken
+===============
+
+Sprint Theme
```

Comment:
@adereis, how about this? https://github.com/apahim/avocado/blob/release_notes_v2/docs/source/38_0.rst